### ISSUE TITLE
Add Oculus Browser to browser list

### ISSUE THREAD

**Issue by claytongulick** [Fyrd/caniuse#4467]
Requesting the addition of the Oculus browser to the browser support list so that we can see what features are possible to implement when developing VR applications for the Oculus Go

**Comment by Fyrd** [comment]
Wasn't able to find any details on this, but I suspect the browser's based on Chromium or something so probably not worth adding.

**Comment by Artyom17** [comment]
Yes, it is Chromium-based browser, but even Edge now is. WebVR and WebXR support is different from the Chromium of the same revision, as well as, other differences in Web APIs could exist. It would be nice to have it added, I can help with the necessary data for that.

**Comment by De-Panther** [comment]
Was there any progress with this since last comment?
@Fyrd @Artyom17

**Comment by snagy** [comment]
Bumping this.  Would love to see the (now) Quest Browser on here, because it does have a unique set of supported APIs like other Chromium-based browsers.

**Comment by munrocket** [comment]
Bumping this.
